tests/eko/io/test_metadata.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+import pytest
+import yaml
+
+from eko.io import bases, metadata, paths
+
+
+def test_metadata(tmp_path, caplog):
```

Comment:
@andreab1997 I guess one should add here a test to check that rotation are actually persistent ... (can't find the issue at the moment)

Reply:
if you can, please (we need to get people involved :upside_down_face: )